### PR TITLE
Update free-programming-books-langs.md

### PR DESCRIPTION
**Pull request by vineetsingh0809**
## What does this PR do?
Add resource(s)

### Description
Added some eBooks of different subject

### Why is this valuable (or not)?
Because this helps in preparing for further studies

### How do we know it's really free?
I have attached a free link to the ebook

### For book lists, is it a book? For course lists, is it a course? etc.
Book

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [J](#j)
 * [Java](#java)
   * [Codename One](#codename-one)
-  * [Java Reporting](#java-reporting)
+  * [Java Reporting](#javareports)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:91]: why?

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1705,7 +1706,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [PHP 5 Power Programming](http://www.informit.com/content/images/013147149X/downloads/013147149X_book.pdf) (PDF)
 * [PHP Apprentice](https://phpapprentice.com) - Andrew Davis *(:construction: in process)*
 * [PHP Best Practices](https://phpbestpractices.org)
-* [PHP Documentor - Documentation](https://docs.phpdoc.org) - phpdoc.org
+* [PHP Documentor - Documentation](https://docs.phpdoc.org) - phpdoc.orgHTML5 Notes for Professionals
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1709]: why?

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -1939,7 +1941,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [SICP in Python](http://www-inst.eecs.berkeley.edu/~cs61a/sp12/book/) (3.2)
 * [Slither into Python: An introduction to Python for beginners](https://web.archive.org/web/20210411065902/https://www.slitherintopython.com/) (3.7) *(:card_file_box: archived)*
 * [Supporting Python 3: An In-Depth Guide](http://python3porting.com) (2.6 - 2.x & 3.1 - 3.x)
-* [Test-Driven Web Development with Python](http://www.obeythetestinggoat.com/pages/book.html) - Harry Percival (3.3 - 3.x)
+* [Test-Driven Web Development with Python](http://www.obeythetestinggoat.com/pages/book.html) - Harry J.W. Percival(3.3 - 3.x)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1944]: Why?

File: books/free-programming-books-langs.md [hunk 4]
```diff
@@ -914,6 +914,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Airbnb CSS-in-JavaScript Style Guide](https://airbnb.io/javascript/css-in-javascript/) - Airbnb
 * [An advanced guide to HTML&CSS](http://learn.shayhowe.com/advanced-html-css/)
 * [Atomic Design](https://atomicdesign.bradfrost.com) - Brad Frost
+* [Building iPhone Apps with HTML, CSS, and JavaScript](https://www.dbooks.org/building-iphone-apps-with-html-css-and-javascript-0596805780/) -  Jonathan Stark(PDF, EPUB, Kindle)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:917]: This appears not to be an authorized distribution. Please see CONTRIBUTING.md guidelines.

**Review by eshellman — CHANGES_REQUESTED** [review]
please explain changes - perhaps you need to start from a fresh fork.

File: books/free-programming-books-langs.md (hunk 3 above)
Comment (davorpa) [reply books/free-programming-books-langs.md:1944]: Seems a bad merge with #6496, or at least start from the same point

File: books/free-programming-books-langs.md (hunk 1 above)
Comment (davorpa) [reply books/free-programming-books-langs.md:91]: Seems a bad merge with #6493, or at least start from the same point

**Comment by vineetsingh0809** [comment]
Hey, sir @eshellman and @davorpa  made the changes you asked about...... Now please review and merge it.

**Comment by davorpa** [comment]
> Hey, sir @eshellman and @davorpa made the changes you asked about...... Now please review and merge it.

Not yet

**Review by davorpa — CHANGES_REQUESTED** [review]
pending changes...

**Comment by davorpa** [comment]
Duplicate of #6496

**Pull request closed without merging**